mozilla_repo_urls/errors.py

```diff
@@ -8,8 +8,13 @@ def __init__(self, url_string) -> None:
 
 
 class UnsupportedPlatformError(RepoUrlsBaseError):
+    @property
+    def supported_platforms(self):
+        return list(self._supported_platforms)
+
     def __init__(self, url_string, platform, supported_platforms) -> None:
```

Comment:
From previous comments, let's update the function signature to get better namings: 

```suggestion
    def __init__(self, url_string, host, supported_hosts) -> None:
```